### DISCUSSION TITLE
Override default config

### DISCUSSION THREAD

**Discussion by rachitpant** [ElMassimo/js_from_routes#32] [Q&A] [answered]
Hello , I am using this library with axios client,  I want to use the `definePathHelper` to define a path manually since this path has a different base url (not the default config) .

i tried something like the following:
```
import { definePathHelper } from "@js-from-routes/axios";
const RESOURCES_ORG_END_POINT = `${process.env.resourcesApiUrl}/v1/orgs`;
export default {
  getOrgs: definePathHelper("get", RESOURCES_ORG_END_POINT),
};
```

However I realized the second param has to be path template relative to a base url  and not an absolute path.
How can i override/ use a different config (baseurl ) to define this  path , while still using the default config in all the other (autogenerated) paths.

**Comment by ElMassimo** [comment, marked as answer]
Hi Rachit!

You should be able to use an absolute URL in `definePathHelper`.

`formatUrl` will [avoid prepending](https://github.com/ElMassimo/js_from_routes/blob/main/packages/client/src/api.ts#L31) the [configured base](https://github.com/ElMassimo/js_from_routes/blob/main/packages/client/src/config.ts#L14-L17) if the provided path is not relative.

**Comment by rachitpant** [comment]
Thanks. Should have looked at the src. Thank you again for this helpful
library.
